tasks/image-segmentation/task.yaml

```diff
@@ -11,6 +11,11 @@ param_groups:
     value: null
     data_type: dataset
     required: true
+  - name: trainedModel
+    description: Model id used for image segmentation.
```

Comment:
Can you improve descriptions so that it is clear which parameters are for training, which for validation, and which are shared by both?